tsdb/chunks/head_chunks_test.go

```diff
@@ -156,24 +156,21 @@ func TestHeadReadWriter_WriteChunk_Chunk_IterateChunks(t *testing.T) {
 
 }
 
-// TestHeadReadWriter_Truncate tests
+// TestChunkDiskMapper_Truncate tests
 // * If truncation is happening properly based on the time passed.
 // * The active file is not deleted even if the passed time makes it eligible to be deleted.
 // * Empty current file does not lead to creation of another file after truncation.
 // * Non-empty current file leads to creation of another file after truncation.
-func TestHeadReadWriter_Truncate(t *testing.T) {
-	hrw, close := testHeadReadWriter(t)
+func TestChunkDiskMapper_Truncate(t *testing.T) {
+	hrw, close := testChunkDiskMapper(t)
 	defer func() {
 		testutil.Ok(t, hrw.Close())
 		close()
 	}()
 
 	timeRange := 0
 	fileTimeStep := 100
-	totalFiles := 7
-	startIndexAfter1stTruncation, startIndexAfter2ndTruncation := 3, 6
-	filesDeletedAfter1stTruncation, filesDeletedAfter2ndTruncation := 2, 5
-	var timeToTruncate, timeToTruncateAfterRestart int64
+	var thirdFileMint, sixthFileMint int64
```

Comment:
Throughout the file, I'd capitalize the "t" here to make clear it's a timestamp, e.g. `thirdFileMinT`, but also `minT` instead of `mint`. I was really confused at first. Is "mint" something the creates coins of some kind? https://en.wikipedia.org/wiki/Mint_(facility) Or is it refering to https://en.wikipedia.org/wiki/Mentha ?

Reply:
I will change for `thirdFileMint` and `sixthFileMint`, but `mint` and `maxt` is used in way too many places :P I will do it for this file though

Reply:
Or leave those `mint` to `minT` changes to a different PR